galley/pkg/metadata/kube/types.go

```diff
@@ -523,6 +523,17 @@ func init() {
 		Converter: converter.Get("identity"),
 	})
 
+	b.Add(kube.ResourceSpec{
+		Kind:      "MeshRbacConfig",
```

Comment:
@ozevren, should we stick with `Mesh` prefix for cluster-scoped resources or adopt k8s convention and use `Cluster`?

Reply:
I had a quick offline chat with @ozevren. I think we want to move towards using `Cluster` prefix for cluster-scoped k8s resources. Mesh-scoped (i.e. multi-cluster) resources ought to be expressed as another layer on top of cluster-scoped resources.

Reply:
Sure, I'll update the PR to rename it to `ClusterRbacConfig`.
/cc @diemtvu @liminw